xacro.cmake

```diff
@@ -40,8 +40,8 @@ MACRO(RUN_XACRO INPUT OUTPUT)
 
   ADD_CUSTOM_COMMAND(
     OUTPUT ${OUTPUT}
-    COMMAND ${_xacro_py}
-    ARGS ${XACRO_OPTIONS} -o ${OUTPUT} ${INPUT}
+    COMMAND ${PYTHON_EXECUTABLE}
```

Comment:
Could you add a FINDPYTHON at the beginning of the macro ?

Reply:
Sure.
Will do if we decide do keep it :)